tests/resources/Vsphere-Util.robot

```diff
@@ -203,6 +203,7 @@ Add Host To VCenter
     \   ${out}=  Run  govc cluster.add -hostname=${host} -username=${user} -dc=${dc} -password=${pw} -noverify=true
     \   ${status}=  Run Keyword And Return Status  Should Contain  ${out}  OK
     \   Return From Keyword If  ${status}
+    \   Sleep  3 minutes
```

Comment:
no that is coming from govmomi, not much to do there AFAICT, but we should not hammer a system that is having problems.